website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -80,6 +80,51 @@ Use this variant only when you need to put interactive elements such as a button
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =800x*)
 
+## Sizes
+
+Our Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium`, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, so AccordionItem sizes are not interchangeable.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+Our Accordion has two container types: `card` and `flush`.
+
+### Card
+
+The `card` variant is the default and recommended type, creating visual separation between Accordion sections and other UI elements. It helps organize content blocks and distinguish between sections within Tabs. Unlike `Cards`/`Tiles` or `Tables`, the `card` variant groups multiple expandable sections for flexible content management.
+
+!!! Do
+
+![image of DO example for the card variant](/assets/components/accordion/accordion-card-type-example.png)
+!!!
+
+### Flush
+
+Use the `flush` variant for tight spaces. For example, use it within other containers like Cards and Flyouts, or in sidebars where space is limited.
```

Comment:
[suggestion] this could be simplified a bit and combined

```suggestion
Use the `flush` variant for tight spaces like within a Card, Flyout, sidebar, or other containers where space is limited.
```